dissect/util/feature.py

```diff
@@ -0,0 +1,66 @@
+import functools
+import os
+from enum import Enum
+from typing import Callable, Optional
+
+
+# Register feature flags in a central place to avoid chaos
+class Feature(Enum):
+    NOVICE = "novice"
```

Comment:
I don't necessarily agree with the feature name "novice", shouldn't it be the other way around? That you guard more advanced features?

Reply:
Ok, used "advanced" instead.